app/src/org/commcare/fragments/connect/ConnectJobIntroFragment.java

```diff
@@ -0,0 +1,74 @@
+package org.commcare.fragments.connect;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.commcare.dalvik.R;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Fragment for showing detailed info about an available job
+ *
+ * @author dviggiano
+ */
+public class ConnectJobIntroFragment extends Fragment {
+    public ConnectJobIntroFragment() {
+        // Required empty public constructor
+    }
+
+    public static ConnectJobIntroFragment newInstance() {
+        return new ConnectJobIntroFragment();
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        ConnectJob job = ConnectJobIntroFragmentArgs.fromBundle(getArguments()).getJob();
+        getActivity().setTitle(job.getTitle());
```

Comment:
can we use a [Android view model](https://developer.android.com/topic/libraries/architecture/viewmodel) to bind the UI here instead ? That will take care of things like restoring data state on configuration changes.